### PR TITLE
Fix target="_blank" vulnerability (and capitalized the H in GitHub)

### PR DESCRIPTION
**Pull request by drepram**
I've updated the social links, fixed the vulnerability[1] it once had. I've also capitalized the H in `GitHub` like how it should be[2], unlike the commonly misspelled `Github`.

[1]: https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/
[2]: https://github.com/keybase/client/issues/10808

PS: I'm a GCI 2019 participant, can this be made into a task?

Thanks,
drepram

### CODE REVIEW

**Comment by drepram** [comment]
Hi, please review the following PR @RishabhJain2018

**Comment by drepram** [comment]
I have done the changes that you have requested. Please take a look.

File: frontend/src/components/navbar/index.js [hunk 1]
```diff
@@ -64,7 +64,11 @@ class Navbar extends Component {
       if (path === "GSoC") {
         return (
           <NavbarItem active={false} key={path}>
-            <Link to="http://gsoc.cloudcv.org" target="_blank">
+            <Link
+              to="http://gsoc.cloudcv.org"
+              target="_blank"
+              rel="noopener noreferrer"
```
Comment (Sanji515) [inline frontend/src/components/navbar/index.js]: @drepram I think we don't need it here as `http://gsoc.cloudcv.org` is a subdomain.

File: frontend/src/components/navbar/index.js [hunk 2]
```diff
@@ -73,7 +77,11 @@ class Navbar extends Component {
       if (path === "GCI") {
         return (
           <NavbarItem active={false} key={path}>
-            <Link to="http://gci.cloudcv.org" target="_blank">
+            <Link
+              to="http://gci.cloudcv.org"
+              target="_blank"
+              rel="noopener noreferrer"
```
Comment (Sanji515) [inline frontend/src/components/navbar/index.js]: Same for here.

File: frontend/src/components/projects/Sidebar.js [hunk 3]
```diff
@@ -48,7 +48,11 @@ class ProjectSidebar extends React.Component {
             {PROJECTS.map((project, index) => {
               return (
                 <li key={index} className="cv-project-list-item">
-                  <Link to={project.link} target="_blank">
+                  <Link
+                    to={project.link}
+                    target="_blank"
+                    rel="noopener noreferrer"
```
Comment (Sanji515) [inline frontend/src/components/projects/Sidebar.js]: Same for here also as project links are also sub-domains.

**Comment by drepram** [comment]
I've done the changes that you've requested. Please take a look.

**Comment by drepram** [comment]
If you can, or at least know someone who can, it might be lovely to make this into a GCI task. Thanks!

**Pull request merged**